populate/treadmill_populate.py

```diff
@@ -0,0 +1,19 @@
+import time
+from populate import default_settings
+from treadmill_pipeline import treadmill
+
+
+limit = 3300  # 55 minutes in seconds
```

Comment:
might just do `60 * 55` for clarity, not important -
but: should there be a way to not have a limit as well e.g. 0